### PR TITLE
Adds confirmation message to Latespawn toggle and adds shutter to south bar window

### LINKED ISSUE #21685: Box: The windows at the bar's entrance and the kitchen don't have shutters

### PR DESCRIPTION
**Pull request by swankcookie**
possible fix for #21097

fixes #21685

### CODE REVIEW

**Comment by ChangelingRain** [comment]
honestly you should just remove it there's no good reason it should exist

**Comment by swankcookie** [comment]
Well it was introduced for some reason.  I personally think it should just tele people to CTF instead.

**Comment by lordpidey** [comment]
Please keep PRs relatively atomic, latejoins have nothing to do with lack of shutters.

**Comment by swankcookie** [comment]
yeah I know, sorry.  I forgot to switch branches when I commited, so i just went for it.

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 1]
```diff
@@ -95775,7 +95775,7 @@ aQc
 aRx
 aQc
 aQc
-aOK
+aQg
```
Comment (xxalpha) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: It's beautiful

Comment (swankcookie) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: hail mapmerger

**Comment by swankcookie** [comment]
@ChangelingRain ready

**Pull request merged**